src/main/java/com/capgemini/domain/Person.java

```diff
@@ -1,19 +1,40 @@
 package com.capgemini.domain;
 
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 import java.util.Date;
 
 public abstract class Person {
+    @NotNull
+    @NotEmpty
     protected String firstName;
```

Comment:
Geldt dit niet alleen voor firstName nu?

Also: is geen rollen commit, misschien in nieuwe branch?